<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormUsers</class>
 <widget class="QWidget" name="FormUsers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>550</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../../usr/share/kademar/utils/cadi/img/bg/cadi.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QStackedWidget" name="pages">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>491</width>
     <height>421</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="software">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>20</y>
       <width>471</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Users</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>411</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>You can manage you user accounts</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="l_logo_blth">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>131</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../../usr/share/kademar/utils/cadi/img/users.png</pixmap>
     </property>
    </widget>
    <widget class="QListWidget" name="list_user">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="b_delUser">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>330</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete User</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/cadi/img/list-remove-user.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/list-remove-user.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="b_addUser">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>190</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Add User</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/cadi/img/list_add_user.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/list_add_user.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="b_modifyUser">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>250</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Modify User</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/cadi/img/user-properties.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/user-properties.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="l_warn">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>420</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>170</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>170</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>115</red>
           <green>120</green>
           <blue>124</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QLineEdit" name="le_user">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>130</y>
       <width>201</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_login">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>180</y>
       <width>201</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_passwd">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>230</y>
       <width>201</width>
       <height>26</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_passwd2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>280</y>
       <width>201</width>
       <height>26</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>42</x>
       <y>133</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>User Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>183</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Account Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>233</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>283</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Repeat Password</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="b_create">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>380</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Accept</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/cadi/img/ok.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/ok.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="b_cancel">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>380</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/cadi/img/enrera.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/enrera.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>471</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Create New User Account</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLineEdit" name="le_modify_user">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>280</y>
       <width>201</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="l_pass">
     <property name="geometry">
      <rect>
       <x>42</x>
       <y>183</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_modify_pass">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>180</y>
       <width>201</width>
       <height>26</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="b_cancel_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>380</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/cadi/img/enrera.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/enrera.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>471</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Modify User Account</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="b_modify">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>380</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Accept</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/cadi/img/ok.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/ok.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
    <widget class="QCheckBox" name="ch_pass">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>191</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Change Pasword</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_user">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>User Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="ch_username">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>240</y>
       <width>241</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Change User Name</string>
     </property>
    </widget>
    <zorder>le_modify_user</zorder>
    <zorder>l_pass</zorder>
    <zorder>le_modify_pass</zorder>
    <zorder>b_cancel_2</zorder>
    <zorder>label_13</zorder>
    <zorder>b_modify</zorder>
    <zorder>ch_pass</zorder>
    <zorder>ch_username</zorder>
    <zorder>l_user</zorder>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../../usr/share/kademar/utils/cadi/img/kademar.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="b_sortir">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../../usr/share/kademar/utils/cadi/img/enrera.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/enrera.png</iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
